### PR TITLE
CSS + Profile

### PR DESCRIPTION
**Pull request by israelmuca**
This PR:
- Fixes the CSS for the Sidebar (font, colors, hover state, etc...)
- Fixes a route problem I introduced, where you were to able to navigate into a protocol
- Added the Profile page content
- Prepared the components to get data from an API.

### CODE REVIEW

File: frontend/src/pages/Dapp/Overview/Index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    </NavLink>
+  );
+};
+
+const Index = () => {
+  return (
+    <main className="grid grid-cols-4 gap-6">
+      {[...Array(5)].map((_, i) => (
+        <ProtocolCard
+          to="uniswap"
```
Comment (Kipitup) [inline frontend/src/pages/Dapp/Overview/Index.js]: would be great to make it more composable:

- create state variable with 'useState' hook for all those infos
- create a function that will query data from ceramic, process it and push it in those state variable.

**Review by Kipitup — APPROVED** [review]
Great work. I added a comment file Overview/Index.js

File: frontend/src/pages/Dapp/Overview/Index.js (hunk 1 above)
Comment (israelmuca) [reply frontend/src/pages/Dapp/Overview/Index.js]: Upgraded!

**Review by israelmuca — COMMENTED** [review]
🚀

**Pull request merged**